samples/MSAppGenerator/ExampleGenerator.cs

```diff
@@ -1,25 +1,34 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using Microsoft.Extensions.DependencyInjection;
 using Microsoft.PowerPlatform.PowerApps.Persistence.Templates;
```

Comment:
nit: I know it was already there, but since you're touching the file you can sort the imports. VS context menu has 'Remove and sort usings' option.

Reply:
I've updated the files in MSAppGenerator to correct this, thanks for the hint on the builtin formatter